scripts/_helper.py

```diff
@@ -10,7 +10,7 @@
 def load_duplicates(duplicates_file_path: Path, ignore: bool = False) -> list[tuple[int, int]]:
     old_duplicates: list[tuple[int, int]] = []
     first = True
-    with open(duplicates_file_path) as duplicates_file:
+    with open(duplicates_file_path, newline='') as duplicates_file:
```

Comment:
AFAICT passing `newline=None` would normalise all newline styles into `\n`. Don't we want that behaviour instead?

Reply:
`newline=''` is explicitly recommended in documentation: https://docs.python.org/3/library/csv.html . The issue with not setting is that there is a \r\n\r per "newline", so basically two onex, which is what Thorsten experienced with his empty lines.

Reply:
Ah, I read the [`open()` docs](), so I thought we want `newline=None`:

> newline determines how to parse newline characters from the stream. It can be `None`, `''`, `'\n'`, `'\r'`, and `'\r\n'`. It works as follows:
> - When reading input from the stream, if `newline` is `None`, universal newlines mode is enabled. Lines in the input can end in `'\n'`, `'\r'`, or `'\r\n'`, and these are translated into `'\n'` before being returned to the caller. If it is `''`, universal newlines mode is enabled, but line endings are returned to the caller untranslated. If it has any of the other legal values, input lines are only terminated by the given string, and the line ending is returned to the caller untranslated.